### DISCUSSION TITLE
Steam doesn't want to installe games...

### DISCUSSION THREAD

**Discussion by IlYAN-FISHERMAN** [Whisky-App/Whisky#634] [Q&A]
Hello guys, i have a problem when i want Install a game on steam with wisky, When I start the download, the download starts, it crashes after a few minutes (the download rate drops to 0 and stays like that)

**Comment by PenguinOnStilts** [comment]
Likewise. In my case the download completes and then hangs right before the Steam install verification phase would normally start. Can even add other installs to the queue, start another, and it'll stall at the same point as well. Thank you!

**Comment by cadenkriese** [comment]
Switching from ethernet to wifi solved this issue for me.

Edit: nevermind it got to 3gb and halted.

**Reply by PenguinOnStilts** [reply]
Yeah, on wifi myself and no dice. Hmmmm. If I were to guess it's some sort of write access issue, but that seems odd. 🤔

**Comment by PenguinOnStilts** [comment]
FYI, does *not* seem resolved in 2.2.1. :)

**Comment by PenguinOnStilts** [comment]
I'm really starting to think this is more about the latest release of the GPTK than Whisky. CrossOver 23.7 does the same thing now for me. :-\

**Comment by PenguinOnStilts** [comment]
Got it, I think! Disable e-sync _and_ m-sync and games will install just fine! Just turn it back on when you're done. :)

My reddit post rabbit hole: https://www.reddit.com/r/macgaming/comments/187ntsl/cannot_download_steam_games_on_either_crossover/

**Comment by AntonyPG** [comment]
I have had this problem most of today, nothing downloading at all. I tried all suggestions I could fix it. I then decided to check the C drive for the virtual machine and took a random guess at deleting everything from the drive_c\users\crossover\Temp directory. This fixed the problem for me, everything downloaded ultra quickly.

**Comment by Glujaz** [comment]
If you are talking about Steam not finalizing the installation (it stays at 98% for me), quitting Steam and reopen it solves the issue. You will find the game installed at relaunch.
